docker/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+# Replace docker args with the script args
+args="$(echo $@ | sed 's/^.*-- //')"
+echo "Args: $args"
```

Comment:
We should print this as a cell in the notebook because of the format compatibility.

Reply:
That's hard to do. I piped it to stderr in the meantime.